pkg/scheduler/ugm/manager.go

```diff
@@ -631,6 +641,21 @@ func (m *Manager) CanRunApp(queuePath, applicationID string, user security.UserG
 	return userCanRunApp && groupCanRunApp
 }
 
+func (m *Manager) HasWildCardApplied(user string, hierarchy []string) bool {
+	m.RLock()
+	defer m.RUnlock()
+	log.Log(log.SchedUGM).Debug("Has wild card user limit applied?",
+		zap.String("user", user),
+		zap.Strings("queue path", hierarchy))
+	userTracker := m.GetUserTracker(user)
+	if userTracker == nil {
+		log.Log(log.SchedUGM).Error("user tracker must be available in userTrackers map",
+			zap.String("user", user))
+		return false
+	}
```

Comment:
This branch is easy to test, let's add test for it to enhance coverage.

Reply:
Filed [Fix codecoverage warnings in UGM](https://issues.apache.org/jira/browse/YUNIKORN-2307) to address this.